### PR TITLE
nspr: generate gcc11 binaries

### PR DESCRIPTION
**Pull request by ericLemanissier**
Specify library name and version:  **nspr/***

This is also a good place to share with all of us **why you are submitting this PR** (specially if it is a new addition to ConanCenter): is it a dependency of other libraries you want to package? Are you the author of the library? Thanks!

---

- [ ] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [ ] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [ ] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [ ] I've tried at least one configuration locally with the [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Review by jwillikers — APPROVED** [review]

**Comment by ericLemanissier** [comment]
Blocked by
- https://github.com/conan-io/conan-center-index/pull/13199

**Comment by ericLemanissier** [comment]
blocks:
- https://github.com/conan-io/conan-center-index/pull/13106

**Review by jwillikers — APPROVED** [review]

**Review by jwillikers — CHANGES_REQUESTED** [review]
Should probably fix the Conan hooks and link to `rt` as necessary.

**Review by jwillikers — DISMISSED** [review]

File: recipes/nspr/all/conanfile.py [hunk 1]
```diff
@@ -1,12 +1,14 @@
...
+from conan.tools.scm import Version
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conans import tools, AutoToolsBuildEnvironment
 import contextlib
 import functools
 import os
 
-required_conan_version = ">=1.36.0"
+required_conan_version = ">=1.52.0"
```
Comment (uilianries) [inline recipes/nspr/all/conanfile.py:11]:
```suggestion
required_conan_version = ">=1.47.0"
```
Version() does not use .major or .minor here, so 1.47.0 is enough

**Review by jwillikers — APPROVED** [review]

**Review by prince-chrismc — APPROVED** [review]

**Pull request merged**